content/plugin/mux/translating-protocol-version-5-to-6.mdx

```diff
@@ -0,0 +1,88 @@
+---
+page_title: 'Plugin Development - Translating Protocol Version 5 to 6'
+description: >-
+  Use the tf5to6server package in terraform-plugin-mux to translate protocol version 5 providers to protocol version 6.
+---
+
+# Translating Protocol Version 5 to 6
+
+The [`tf5to6server`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-mux/tf5to6server) package enables translating a [protocol version 5 provider server](../how-terraform-works#protocol-version-5) into a [protocol version 6 provider server](../how-terraform-works#protocol-version-6).
+
+Use this package to:
+
+* Migrate individual resources and data sources from [terraform-plugin-sdk/v2](../sdkv2/sdk-intro) to [terraform-plugin-framework](../framework) over time, while taking advantage of [protocol version 6](../how-terraform-works#protocol-version-6) features in terraform-plugin-framework.
```

Comment:
We just linked to the protocol 6 stuff, so we probably don't need it again here so soon. 

```suggestion
* Migrate individual resources and data sources from [terraform-plugin-sdk/v2](../sdkv2/sdk-intro) to [terraform-plugin-framework](../framework) over time, while using protocol version 6 features in terraform-plugin-framework.
```